#/usr/bin/env python

PACKAGE = "laser_daten"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ek", double_t, 0, "Punkte die zur linie gehoeren", 0.8, 0, 1)
gen.add("add_ek", double_t, 0, "Punkte die zur linie gehoeren strenger", 0.001, 0, 1)
gen.add("abstand_punkte_red", double_t, 0, "Abstand der Punkte reduktionsfilter", 0.0001, 0, 1)
gen.add("zul_einzelabstand", double_t, 0, "Einzelabstand der punkte zueinanderr", 0.005, 0, 1)
gen.add("fixed_frame", str_t, 0, "rviz: fixed frame", "/map")

exit(gen.generate(PACKAGE, "laser_daten", "myParam"))
